src/main/java/daos/GameDAO.java

```diff
@@ -50,35 +78,35 @@ public IDTO getGameModel(IDTO dto) {
      * @param dto
      */
     @Override
-    public void updateGame(IDTO dto) {
-        try {
-            Statement st = Database.getConnection().createStatement();
-        } catch (SQLException e) {
-            e.printStackTrace();
-        }
+    public void updateGame(GameDTO dto) throws SQLException {
+        Statement stmt = Database.getConnection().createStatement();
+        String sql = "UPDATE GAME set STATE = " + dto.getState()
+                + "where GAMEID=" + dto.getGameID() + ";";
```

Comment:
good job using update I was thinking about deleting and then adding.  This is better!